aws-backend/amplify/backend/function/fellowendpoint/src/app.js

```diff
@@ -1,10 +1,12 @@
-const AWS = require('aws-sdk')
-const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware')
-const bodyParser = require('body-parser')
-const express = require('express')
+const AWS = require('aws-sdk');
+const awsServerlessExpressMiddleware = require('aws-serverless-express/middleware');
+const bodyParser = require('body-parser');
+const express = require('express');
+const fetch = require('node-fetch');
 
 AWS.config.update({ region: process.env.TABLE_REGION });
 
+const BASE_API_URL = 'https://api.github.com/graphql';
```

Comment:
It might be useful to specify in the variable name that this is the GraphQL API, so as to avoid mixups with the REST API.